.gitignore

```diff
@@ -27,13 +27,6 @@
 /DOCS/man/mpv.pdf
 /DOCS/man/mpv.toc
```

Comment:
You could remove all of these since meson only does out of tree builds (plus a good chunk don't even exist anymore). Ditto for the makefile entry below.

Reply:
I figured that was best as a separate commit or even a separate PR. AFAICT, only `/subprojects` is really needed.

Reply:
Well it's kind of a mix. Some of these files really will exist if you do a build in the tree with waf (like `config.h`, `*.o`, and so on).

Reply:
I guess I meant post-waf. I suppose I could just do that in this commit as well though, since they're all meaningless without waf.